src/components/RecipesList.js

```diff
@@ -36,21 +43,31 @@ class RecipesList extends React.Component {
         let recipes = data.map((recipe) => {
           return {id:recipe.id, title:recipe.title, image:recipe.image};
         });
+        this.setState({recipes: recipes})
+        this.setState({ingredients: []})
```

Comment:
Can this not all be folded into one setState instruction

Reply:
Resolved